src/themes/teams/components/Divider/dividerStyles.ts

```diff
@@ -33,43 +33,36 @@ const dividerStyles: IComponentPartStylesInput = {
   }): ICSSInJSStyle => {
     const { children, size, type, important, content } = props
     return {
-      marginTop: pxToRem(5 + size * 7.5),
-      marginBottom: pxToRem(5 + size * 7.5),
-      fontWeight: 400,
+      color: variables.textColor,
+      display: 'flex',
+      alignItems: 'center',
+      ...(type === 'primary' && {
+        color: variables.primaryColor,
```

Comment:
does this set color twice then?

Reply:
No, if the type is primary, the second will override the first definition.